samples/word/25-paragraph/get-word-count.yaml

```diff
@@ -0,0 +1,131 @@
+order: 8
+id: word-paragraph-get-word-count
+name: Get word count
+description: Counts how many times a word or term appears in the document.
+host: WORD
+api_set:
+    WordApi: 1.1
+script:
+    content: |
+        $("#count").click(() => tryCatch(parseText));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function parseText() {
+          await Word.run(async (context) => {
+            let paragraphs = context.document.body.paragraphs;
+            paragraphs.load("text");
+            await context.sync();
+
+            let text = [];
+            paragraphs.items.forEach((item) => {
+              let paragraph = item.text.trim();
+              if (paragraph) {
+                paragraph.split(" ").forEach((term) => {
+                  let currentTerm = term.trim();
+                  if (currentTerm) {
+                    text.push(currentTerm);
+                  }
+                });
+              }
+            });
+
+            let makeTextDistinct = new Set(text);
+            let distinctText = Array.from(makeTextDistinct);
+
+            for (let i = 0; i < distinctText.length; i++) {
+              await searchAndCount(distinctText[i], context);
+            }
+          });
+        }
+
+        async function searchAndCount(searchTerm: string, context: Word.RequestContext) {
+          let results = context.document.body.search(searchTerm, { matchCase: true, matchWholeWord: true });
+          results.load("text");
+
+          await context.sync();
+
+          let length = 0;
+          if (results && results.items && results.items.length > 0) {
+            length = results.items.length;
+          }
+
+          console.log("Search term: " + searchTerm + " => Count: " + length);
+        }
+
+        async function setup() {
+          await Word.run(async (context) => {
+            // Let's insert a couple of paragraphs to illustrate the point.
+            context.document.body.clear();
+            context.document.body.insertParagraph("Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.", "Start");
+            context.document.body.insertParagraph("To make your document look professionally produced, Word provides header, footer, cover page, and text box designs that complement each other. For example, you can add a matching cover page, header, and sidebar. Click Insert and then choose the elements you want from the different galleries.", "End");
+
+            await context.sync();
+
+            context.document.body.paragraphs.getFirst().alignment = "Left";
```

Comment:
What are these two lines for? I think paragraphs in Word are left-aligned by default.